Makefile

```diff
@@ -172,9 +171,36 @@ docker-push-manifest: ## Push the fat manifest docker image.
 	## Minimum docker version 18.06.0 is required for creating and pushing manifest images.
 	docker manifest create --amend $(CONTROLLER_IMG):$(TAG) $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~$(CONTROLLER_IMG)\-&:$(TAG)~g")
 	@for arch in $(ALL_ARCH); do docker manifest annotate --arch $${arch} ${CONTROLLER_IMG}:${TAG} ${CONTROLLER_IMG}-$${arch}:${TAG}; done
-	docker manifest push --purge ${CONTROLLER_IMG}:${TAG}
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${CONTROLLER_IMG}:$(TAG)"'@' ./config/default/manager_image_patch.yaml
+	MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) $(MAKE) set-manifest-image
+
+.PHONY: set-manifest-image
+set-manifest-image:
+	$(info Updating kustomize image patch file for manager resource)
+	sed -i'' -e 's@image: .*@image: '"${MANIFEST_IMG}:$(MANIFEST_TAG)"'@' ./config/default/manager_image_patch.yaml
+
+## --------------------------------------
+## Release
+## --------------------------------------
+
+RELEASE_TAG := $(shell git tag --contains HEAD | tail -n 1)
```

Comment:
Would `git describe` be better here to ensure the tag is an annotated tag?

Reply:
That returns a different value though, for example
```
$ git describe --tags --contains 961efcc224d6c5b278ecf19aee8512e8c5a98283
v0.1.9^0
```

I'd have to parse and strip everything after `^`

Reply:
I was thinking `git describe --abbrev=0`

Reply:
sgtm, I can switch it up